client/src/infographic/BubbleMenu.js

```diff
@@ -52,11 +52,20 @@ const BubbleMenu = ({ items, active_item_id }) => {
             >
               <div className="bub-item">
                 <strong className="title">{title}</strong>
-                <div
-                  className="bub-svg"
-                  title={description}
-                  dangerouslySetInnerHTML={{ __html: svg_content }}
-                />
+                <div className="bub-svg" title={description}>
+                  {(() => {
+                    const SVG = (svg_content && svg_content.svg) || null;
+                    return (
+                      SVG && (
+                        <SVG
+                          alternate_color={false}
+                          width="90%"
+                          direct_render
```

Comment:
Is direct render because `.bub-svg` has some specific selector rule that doesn't expect the `g` added by the icon wrapper? I'm not a fan of the `direct_render` prop because it, not super obviously, disables other props like rotating. Also makes it harder to write other selectors targeting the inner content of icons if they don't have a consistent structure.

Could just change the `.bub-svg` selector here. Better would probably be giving the icons a prop like `inner_class_name` or something (probably to add to the `g`?), so that consumers can get a way to optionally target the svg content of an icon without having to hard code a selector against the implementation of the icon components.